google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/CreateInstanceRequest.java

```diff
@@ -50,7 +51,7 @@
  * // Development instance:
  * CreateInstanceRequest smallProdInstanceRequest = CreateInstanceRequest.of("my-dev-instance")
  *   .setType(Type.DEVELOPMENT)
- *   .addCluster("cluster1", "us-east1-c", 1, StorageType.SSD);
+ *   .addCluster("cluster1", "us-east1-c", 0, StorageType.SSD);
```

Comment:
@igorbernstein2 I hope setting server node to 0 is ok, considering [proto docs](https://github.com/googleapis/googleapis/blob/master/google/bigtable/admin/v2/instance.proto#L65-L67) says server nodes should be unset.

Reply:
I think that looks a bit too weird. I think it would be better to add a separate method "addDevelopmentCluster(clusterId, zoneId, storageType)

Reply:
Updated with the same; PTAL